cloud/wfm/src/result/UrlResult.ts

```diff
@@ -0,0 +1,18 @@
+// NOTE: browserify can provide an implementation for node's url module
+import * as nodeUrl from 'url';
+import {Result} from './index';
+
+/**
+ * Represents a {@link Result} that can be accessed via a URL
+ */
+export default class UrlResult implements Result {
```

Comment:
Sadly everything needs to be offline enabled. We cannot relay on any urls. I totally get you here, but I do think that we just need only object result for the moment to avoid confusion.